daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -8,26 +8,133 @@ weight: 2000
 
 This article describe how to deploy services each with an unique application ID, so that other services can discover and call endpoints on them using service invocation API.
 
+## Example:
+
+There are two services called order processing service and checkout service. Dapr used in order processing service is used to invoke checkout service. Both order processing service and checkout service have dapr side cars.
```

Comment:
```suggestion
The below code examples loosely describe an application that processes orders. In the example, there are two services - an order processing service and a checkout service. Both services have Dapr sidecars and the order processing service uses Dapr to invoke the checkout method in the checkout service.
```

Reply:
@amulyavarote - Reworded this a bit. Thoughts?

Reply:
Looks good!